pyshtools/shclasses/shgravcoeffs.py

```diff
@@ -806,6 +810,89 @@ def mass(self):
         """
         return self.gm / _G.value
 
+    @property
+    def center_of_mass(self):
+        """Return coordinates of the center of mass of the planet in metres.
+
+        This method will return cartesian coordinates of the center of mass with respect
+        to the coordinate system of the spherical harmonic coefficients.
+
+        Returns
+        -------
+        x, y, z : float
+            Cartesian coordinates of the center of mass, in metres.
+        """
+        coeffs = self.convert(normalization='unnorm',
+                csphase=1, lmax=1).coeffs
+
+        x_cm = coeffs[1, 1, 1] * self.r0
```

Comment:
I think that the formulas for x_cm and y_cm are switched. coeffs[0,1,1] is C11 and coeffs[1,1,1] is S11

Reply:
Oops, I agree!